surf_mvp_coordinatable_alert/Code/Configurator/module_configurator.swift.liquid

```diff
@@ -0,0 +1,28 @@
+//
+//  {{ module_info.file_name }}
+//  {{ module_info.project_name }}
+//
+//  Created by {{ developer.name }} on {{ date }}.
+//  Copyright © {{ year }} {{ developer.company }}. All rights reserved.
+//
+
+import UIKit
+
+final class {{ module_info.name }}AlertConfigurator {
+
+    // MARK: - Internal methods
+
+    func configure() -> ({{ module_info.name }}Controller, {{ module_info.name }}ModuleOutput) {
+
+        // TODO: Change title and message
```

Comment:
Maybe do this now?

Reply:
This TODO must be resolved in project code for every alert module.